### DISCUSSION TITLE
What is the process for submitting a new Feature Type?

### DISCUSSION THREAD

**Discussion by julcnx** [openstreetmap/iD#9010] [Q&A] [answered]
What is the process for submitting a new Feature Type?

I can foresee a few use-cases with a common combination of tags that would be useful for new mappers.

**Comment by tordans** [comment, marked as answer]
@cmoffroad https://github.com/openstreetmap/iD/issues/new/choose and https://github.com/openstreetmap/iD#participate guide you to a few places that explain how and where to contribute.

It sounds like you are referring to presets, which would be part of https://github.com/openstreetmap/id-tagging-schema.
